kernel/config-5.15.x-aarch64

```diff
@@ -1753,7 +1753,6 @@ CONFIG_OF_KOBJ=y
 CONFIG_OF_DYNAMIC=y
 CONFIG_OF_ADDRESS=y
 CONFIG_OF_IRQ=y
-CONFIG_OF_NET=y
```

Comment:
Is this removed line expected?

Reply:
It's legit, CONFIG_OF_NET has been removed in 5.15.27
https://github.com/torvalds/linux/commit/e330fb14590c5c80f7195c3d8c9b4bcf79e1a5cd